src/components/PasswordAlert.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Snackbar from '@mui/material/Snackbar';
+import MuiAlert from '@mui/material/Alert';
+
+const Alert = React.forwardRef(function Alert(props, ref) {
+  return <MuiAlert elevation={6} ref={ref} variant="filled" {...props} />;
+});
+
+export default function PasswordAlert(props) {
+  const { anchorOrigin, duration, message, open, setOpen, severity } = props;
+
+  const handleClose = (event, reason) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+
+    setOpen(false);
+  }
+
+  return (
+    <div>
+      <Snackbar
+        open={open}
+        anchorOrigin={anchorOrigin}
+        autoHideDuration={duration}
+        onClose={handleClose}>
+        <Alert
+          open={open}
+          onClose={handleClose}
+          severity={severity}
+          sx={{width: '100%'}}>
+          {message}
+        </Alert>
+      </Snackbar>
+
+    </div>
+  );
+}
```

Comment:
Is the main purpose of the Alert component adding color to the Snackbar? I am wondering if we should roll the Alert into the Snackbar component. Right now the Snackbar is black no matter what it is used for because it was easier for me to get the component started without having to think too much about the colors.

Reply:
The purpose is to have the color and the icon.

I tried to alter your component to pass the Alert in as a prop but I couldn't get it to work, and making a separate component was faster.

Reply:
Why pass in `Alert` as a prop to `SnackBar`? You could have modified `SnackBar.jsx` and defined `Alert` in there and have all the SnackBars have alert colors. 

I think we should change the name of this component to StyledSnackBar, delete SnackBar.jsx, and then use this component going forward.